### ISSUE TITLE
L W [P#M 0.0, P#0.0] crashes

### ISSUE THREAD

**Issue by CapacitorSet** [mbuesch/awlsim#5]
Trying to run `L W [P#M 0.0, P#0.0]` results in a crash, whereas the expected behaviour is to fail gracefully.

Stack trace:

```
AwlSimServer: Shutting down.
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 1207, in <module>
    sys.exit(AwlSimServer._execute())
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 207, in _execute
    server.runFromEnvironment(env)
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 309, in runFromEnvironment
    self.run()
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 1036, in run
    self.__handleCommunicationBlocking()
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 900, in __handleCommunicationBlocking
    self.__handleCommunication()
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 893, in __handleCommunication
    self.__handleSocketComm(rlist)
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 868, in __handleSocketComm
    self.__handleClientComm(self.__sock2client[sock])
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 858, in __handleClientComm
    handler(self, client, msg)
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 637, in __rx_RUNSTATE
    self.setRunState(self.STATE_RUN)
  File "/[...]/awlsim/awlsim/coreserver/server.py", line 335, in setRunState
    self.__sim.startup()
  File "/[...]/awlsim/awlsim/core/main.py", line 42, in profiled_wrapper
    func(self, *args, **kwargs)
  File "/[...]/awlsim/awlsim/core/main.py", line 55, in awlSimErrorExtension_wrapper
    func(self, *args, **kwargs)
  File "/[...]/awlsim/awlsim/core/main.py", line 190, in startup
    self.cpu.startup()
  File "/[...]/awlsim/awlsim/core/cpu.py", line 846, in startup
    self.prog.build()
  File "/[...]/awlsim/awlsim/core/cpu.py", line 317, in build
    self.__detectMnemonics()
  File "/[...]/awlsim/awlsim/core/cpu.py", line 177, in __detectMnemonics
    optrans.translateFromRawInsn(rawInsn)
  File "/[...]/awlsim/awlsim/core/optrans.py", line 671, in translateFromRawInsn
    opDesc = self.translateOp(rawInsn, rawOps)
  File "/[...]/awlsim/awlsim/core/optrans.py", line 627, in translateOp
    self.__translateAddressOperator(opDesc, rawOps[1:])
  File "/[...]/awlsim/awlsim/core/optrans.py", line 345, in __translateAddressOperator
    self.__translateIndirectAddressing(opDesc, rawOps[1:])
  File "/[...]/awlsim/awlsim/core/optrans.py", line 299, in __translateIndirectAddressing
    assert(offsetOpDesc.fieldCount == end)
AssertionError
```

**Comment by mbuesch** [comment]
That's a bug, indeed.
Thanks for the report.

**Comment by mbuesch** [comment]
Fixed in f2d00a0409854ed0ae17c03a1590700ddd5d139b
The fix will be included in awlsim-0.46

Thanks for the bug report.
